src/hyperlight_host/src/hypervisor/hypervisor_handler.rs

```diff
@@ -64,6 +64,7 @@ pub(crate) struct HypervisorHandler {
     communication_channels: HvHandlerCommChannels,
     configuration: HvHandlerConfig,
     execution_variables: HvHandlerExecVars,
+    hypervisor: Option<Arc<Mutex<Box<dyn Hypervisor>>>>,
```

Comment:
I don't think you need to add this to the hypervisor handler struct, since it is only ever used from inside the hv handler thread; just storing the hypervisor in a local created inside the thread should be fine.

Reply:
It also then doesn't need to be Arc/Mutex; it can just be `Option<Box<dyn Hypervisor>>`.

Reply:
Fixed! Thanks Lucy for the help ✅